source/_integrations/denonavr.markdown

```diff
@@ -119,4 +124,13 @@ A comprehensive list of telnet protocol commands is [also available](http://asse
 | `entity_id`            |       no | Name of entity to send command to. For example `media_player.marantz`|
 | `command`              |       no | Command to send to device, e.g.,  `/goform/formiPhoneAppDirect.xml?VSMONI2`|
 
+#### Service `denonavr.set_dynamic_eq`
+
+Enable or disable DynamicEQ setting.
+
+| Service data attribute | Optional | Description                                          |
+| ---------------------- | -------- | ---------------------------------------------------- |
+| `entity_id`            |       no | Name of entity to send command to. For example `media_player.marantz`|
```

Comment:
This is an entity service, meaning entity_id is optional.

Reply:
Done 👍